src/styles/blocks/nav.scss

```
@@ -0,0 +1,47 @@
+.nav__list {
+  line-height: 60px;
+  list-style: none;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  margin: auto;
+}
+
+.nav__links {
```

See checklist item 9
https://github.com/mate-academy/layout_catalog/blob/master/checklist.md
```suggestion
.nav {
  &__list {
    line-height: 60px;
    list-style: none;
    display: flex;
    justify-content: flex-end;
    align-items: center;
    margin: auto;
  }
  
  &__links {
```